spec/features/api_access_spec.rb

```diff
@@ -7,12 +7,28 @@ def response_json
   end
 
   describe "api json export" do
-    it 'routes to the all action on default' do
+    it 'exports all organizations as json by default' do
       create(:organization, name: "organization foo")
       create(:organization)
-      get "/api"
+      get "/api.json"
       expect(response_json.size).to eq(2)
       expect(response_json[0]['name']).to eq("organization foo")
     end
   end
+
+  describe "api markdown export" do
+    it 'spec_name' do
+      create(:organization, name: "organization docx")
+      get "/api.markdown"
+      expect(response.body).to include("# organization docx")
+    end
+  end
+
+  describe "api docx export" do
+    it 'spec_name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.